frontends/api/src/clients.ts

```diff
@@ -24,7 +24,9 @@ import {
 
 import axiosInstance from "./axios"
 
-const { MITOL_API_BASE_URL } = APP_SETTINGS
+const MITOL_API_BASE_URL = (<any>process).browser
+  ? process.env.NEXT_PUBLIC_MITOL_API_BASE_URL
+  : process.env.MITOL_API_BASE_URL
```

Comment:
I believe this can just be `const MITOL_API_BASE_URL = NEXT_PUBLIC_MITOL_API_BASE_URL`.